src/app/groups/containers/group-log-view/group-log-view.component.html

```diff
@@ -55,12 +55,19 @@
                 <ng-template #iconSection>
                   <i class="ph-duotone item-icon" [ngClass]="{
                     'ph-flag-checkered': rowData.activityType === 'result_started',
-                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer'
+                    'ph-floppy-disk-back': rowData.activityType === 'saved_answer' || rowData.activityType === 'current_answer'
                   }"></i>
                 </ng-template>
               </div>
               <div class="item-title">
                 <p class="item-title-caption">{{ rowData.activityType | logActionDisplay }}</p>
+                <a
+                  class="alg-link"
+                  [routerLink]="rowData.item | rawItemRoute | withAnswer: { id: rowData.answerId } | url"
+                  *ngIf="rowData.allowToViewAnswer && rowData.activityType === 'saved_answer' && rowData.answerId"
```

Comment:
Sorry, I want to change a bit the behavior (change my spec): this should also be displayed in case of a "submission"

Reply:
Fixed